xclim/sdba/measures.py

```diff
@@ -442,3 +442,68 @@ def _scorr(
 scorr = StatisticalPropertyMeasure(
     identifier="Scorr", aspect="spatial", compute=_scorr, allowed_groups=["group"]
 )
+
+
+def _taylordiagram(
+    sim: xr.DataArray,
+    ref: xr.DataArray,
+    dim: str = "time",
+    group: str | Grouper = "time",
+) -> xr.DataArray:
+    """Taylor diagram.
+
+    Compute the respective standard deviations of a simulation and a reference array, as well as the Pearson
+    correlation coefficient between both, all necessary parameters to plot points on a Taylor diagram.
+
+    Parameters
+    ----------
+    sim : xr.DataArray
+        data from the simulation (a time-series for each grid-point)
+    ref : xr.DataArray
+        data from the reference (observations) (a time-series for each grid-point)
+    dim : str
+        Dimension across which the correlation and standard deviation should be computed.
+    group : str
+        Compute the property and measure for each temporal groups individually.
+        Currently not implemented.
+
+
+    Returns
+    -------
+    xr.DataArray, [measures]
+        DataArray containing the correlation coefficient and standard deviations as coordinates.
+    """
+    corr = xr.corr(sim, ref, dim=dim)
+
+    ref_std = ref.std(dim=dim, skipna=True, keep_attrs=True)
+    sim_std = sim.std(dim=dim, skipna=True, keep_attrs=True)
+
+    coords = {"taylor_params": ["ref_std", "sim_std", "corr"]}
+    dims = ["taylor_params"]
+
+    for dim in ref_std.dims:
```

Comment:
We want the dims to keep the attrs they had in the inputs, not just the value.

Reply:
Also, now I am thinking that we need the coords that aren't dims to also be conserved ( like `rotated_pole`, `lat` and `lon` for a input in `rlat`-`rlon`